mava/evaluator.py

```diff
@@ -284,7 +291,11 @@ def make_eval_fns(
 ) -> Tuple[EvalFn, EvalFn]:
     """Initialise evaluator_fn."""
     # Check if win rate is required for evaluation.
-    log_win_rate = config.env.env_name in ["HeuristicEnemySMAX", "LearnedPolicyEnemySMAX"]
+    log_win_rate = config.env.env_name in [
+        "HeuristicEnemySMAX",
+        "LearnedPolicyEnemySMAX",
+        "Gigastep",
+    ]
```

Comment:
Can we make this a config option and put it in all the envs configs so that we no longer need to update this list :smile: